go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
```

Comment:
It's not clear why some dependencies are marked as `indirect`. Review if all indirect dependencies are indeed required to simplify the module and reduce potential vulnerabilities from unused packages.